Makefile

```diff
@@ -128,6 +139,17 @@ dumpvar:
 	fi
 	@echo "$(VAR) is '$($(VAR))'"
 
+# eslint ftw
+ESLINT = ./node_modules/.bin/eslint
+$(ESLINT):
+	$(RUN_NPM_INSTALL)
+
+check:: check-eslint
+
+.PHONY: check-eslint
+check-eslint: $(ESLINT)
+	@./node_modules/.bin/eslint ./
```

Comment:
Right, it's overkill in that case. What bothers me really is the duplication of `./node_modules/.bin/eslint`. Can we avoid that?